WcaOnRails/app/models/competition.rb

```diff
@@ -696,6 +696,11 @@ def in_progress?
     !results_posted? && (start_date..end_date).cover?(Date.today)
   end
 
+  def uses_cutoff?
+    cutoff = competition_events.find { |ce| ce.rounds.find { |r| r.cutoff } }
+    !cutoff.nil?
+  end
```

Comment:
Using `any?` would directly result in a boolean value =)
```ruby
def uses_cutoff?
  competition_events.any? { |ce| ce.rounds.any?(&:cutoff) }
end
```

Reply:
Nice :)